mysql-test/suite/innodb/t/bug84563.test

```diff
@@ -0,0 +1,35 @@
+--source include/have_innodb.inc
+
```

Comment:
As you are using an additional connection add
```
--source include/count_sessions.inc
...
--source include/wait_until_count_sessions.inc
```

Reply:
fixed